src/main/common/router/defendants.ts

```diff
@@ -27,4 +28,12 @@ export class Defendants {
   static getCurrentIndex (res: express.Response): number {
     return res.locals.user.legalClaimDraft.defendants.length - 1
   }
+
+  static getCurrentDefendantName (res: express.Response): string {
+    const defendants = res.locals.user.legalClaimDraft.defendants
+    const defendantDetails = defendants[Defendants.getCurrentIndex(res)].defendantDetails
+    const isIndividual = defendantDetails.type.value === PartyTypes.INDIVIDUAL.value
+    const title = defendantDetails.title != null ? defendantDetails.title + ' ' : defendantDetails.title
```

Comment:
please void using + operator to combine strings. Try something like `${var1} ${var2}` to format a string.
Don't append any space here rather format your string in next statement.

```return isIndividual ? `${title} ${defendantDetails.fullName}`   : defendantDetails.organisation```